### PR TITLE
main/: url and license fixes

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by TBK** [comment]
main/sircbot 's url is also incorrect, should be https://git.alpinelinux.org/hosted/sircbot/. @maxice8 do you want to add it or should a make a separate PR?

**Comment by maxice8** [comment]
On Thu, Jun 27, 2019 at 07:24:39AM -0700, TBK wrote:
> main/sircbot 's url is also incorrect, should be https://git.alpinelinux.org/hosted/sircbot/. @maxice8 do you want to add it or should a make a separate PR?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/9150#issuecomment-506368363

Feel free to make a separate PR i'm just fixing as i walk along https://repology.org/repository/alpine_edge/problems

File: main/libspf2/APKBUILD [hunk 1]
```diff
@@ -3,14 +3,13 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libspf2
 pkgver=1.2.10
-pkgrel=3
+pkgrel=4
 pkgdesc="Sender Policy Framework library, a part of the SPF/SRS protocol pair."
-url="http://www.libspf2.org"
+url="https://wiki.gnome.org/Projects/Libsecret"
```
Comment (tcely) [inline main/libspf2/APKBUILD]: Incorrect project URL

File: main/libspf2/APKBUILD [hunk 2]
```diff
@@ -19,33 +18,31 @@ source="http://www.libspf2.org/spf/$pkgname-$pkgver.tar.gz
...
 prepare() {
-	cd "$_builddir"
-	update_config_sub || return 1
+	cd "$builddir"
+	update_config_sub
 	rm -r src/libreplace
 	for i in $source; do
 		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+		*.patch) msg $i; patch -p1 -i "$srcdir"/$i ;;
```
Comment (tcely) [inline main/libspf2/APKBUILD]: Use `default_prepare`?

**Comment by maxice8** [comment]
uhhh

**Comment by maxice8** [comment]
wtf github ?

**Pull request closed without merging**